v1/plugin/flags.go

```diff
@@ -0,0 +1,54 @@
+package plugin
+
+import "github.com/urfave/cli"
+import "time"
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	// If PingTimeoutDuration was provided we set it
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
```

Comment:
I actually have two minds on this.  I don't like the fact that we inverted the order in the framework (0:debug, ...5:panic).  In hindsight, I think we should have followed the standard where increasing the log level increases verbosity.  While it could be a bit painful I think we should discuss migrating to this standard.  It would allow us to use the common `-vvvv` switch where the number of verbose flags are used to increment the log-level.

See http://unix.superglobalmegacorp.com/Net2/newsrc/sys/syslog.h.html.